Octokit/Models/Response/ResourceRateLimits.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Diagnostics;
+using System.Globalization;
+
+using Octokit.Helpers;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class ResourceRateLimits
```

Comment:
Per my [previous suggestion](https://github.com/octokit/octokit.net/pull/848/files#r35466147), I think this should be `ResourceRateLimit` and the property types should be `RateLimit`